compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -488,7 +488,14 @@ convertModule lfVersion pkgMap stablePackages isGenerated file x depOrphanModule
     templates <- convertTemplateDefs env
     exceptions <- convertExceptionDefs env
     interfaces <- convertInterfaces env (eltsUFM (cm_types x))
-    pure (LF.moduleFromDefinitions lfModName (Just $ fromNormalizedFilePath file) flags (types ++ templates ++ exceptions ++ definitions ++ interfaces ++ depOrphanModules))
+    let defs =
+            types
```

Comment:
I got a bit tired of the super long line, so I factored this out. Please let me know if this is the style you'd prefer for splitting the expression into lines, or if you prefer operators at the end or something different.